WordPress/wordpress-playground
# Fix: Import & Export from Github causes reloading the playground even before accept this step.

## Motivation for the change, related issues

Issue #1902 

## Implementation details

Import & Export from Github causes reloading the playground even before accept this step.
I know that it is because of adding new params in the URL.
So I kept functionality to fire those modal via URL, but I also added logic to open those modals without adding params.

## Testing Instructions (or ideally a Blueprint)
1. open http://127.0.0.1:5400/website-server/
2. Go to playground settings
3. Click 3 dots next to "Homepage" button
4. Click export or import from GitHub
5. Modal should appear without reloading

Also you can check, that opening on slug is still working (but with reloading on closing popup):
http://127.0.0.1:5400/website-server/?modal=github-import
